elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/guarantee/GuaranteeService.java

```diff
@@ -17,11 +17,16 @@
 
 package org.apache.shardingsphere.elasticjob.lite.internal.guarantee;
 
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.recipes.locks.InterProcessMutex;
```

Comment:
We have decoupled registry center since 3.0.2. Using curator here directly may not acceptable.